### ISSUE TITLE
Unable to use sudo command

### ISSUE THREAD

**Issue by thomascychan** [ot4i/ace-docker#54]
When I logged in to the bash console as the default "aceuser", the following first line was displayed.

> To run a command as administrator (user "root"), use "sudo <command>".

However, when I tried to run "sudo su - mqm", I got the following exception:

> sudo: effective uid is not 0, is /usr/bin/sudo on a file system with the 'nosuid' option set or an NFS file system without root privileges?

Any ideas why I can't run the sudo command?

**Comment by IBMRob** [comment]
What image are you running this command on? 

If I use the prebuild version from docker hub I don't see this error:

```
Bergkamp-stable$ docker run -it --entrypoint=/bin/bash ibmcom/ace-mq -i
(ACE_11:)root@726eb778b4a4:/home/aceuser# sudo su - mqm
No directory, logging in with HOME=/
$ id
uid=999(mqm) gid=999(mqm) groups=999(mqm)
$ exit
```

If i run the non mq version I do see the warning about sudo but this image does not include an mqm user so would expect the sudo command to fail.

**Comment by thomascychan** [comment]
I followed the instructions to first build the ace-only image, and after that I built the ace-mqclient image. I was trying to update the "mqclient.ini" file under "/var/mqm" to add the "CertificateLabel=" line, but everything is owned by "mqm" and the "aceuser" is not part of that group. The "id" command below does show "aceuser" as part of the "sudo" group but it can't run sudo commands.

`[docker@reg2lxap0034 ubuntu]$ docker exec -it aceserver /bin/bash`
`To run a command as administrator (user "root"), use "sudo <command>".
See "man sudo_root" for details.`
`(ACE_11:)aceuser@c64d72f45074:~$ id`
`uid=1000(aceuser) gid=1001(aceuser) groups=1001(aceuser),27(sudo),1000(mqbrkrs)`
`(ACE_11:)aceuser@c64d72f45074:~$ sudo su - mqm`
`sudo: effective uid is not 0, is /usr/bin/sudo on a file system with the 'nosuid' option set or an NFS file system without root privileges?`

**Comment by thomascychan** [comment]
FYI, my objective is to use the "mqKeyRepository" property in the server.conf.yaml file to set up an SSL connection to a remote MQ server.  In the "ubuntu/installMQClient.sh" script I had to add the "ibmmq-gskit" debian package to allow me to run the "runmqakm" command to create a kdb for my MQ client.  In order to temporary circumvent the "sudo su - mqm" command issue, I had to add "usermod -a -G mqm aceuser" in the script file so that I could update the "mqclient.ini" file.

It would still be nice to be able to run the "sudo" command.
